src/main/frontend/handler/common.cljs

```diff
@@ -28,17 +29,20 @@
           (gobj/get js/window "workerThread")
           (gobj/get js/window.workerThread "getChangedFiles"))
      (->
-      (p/let [files (js/window.workerThread.getChangedFiles (util/get-repo-dir repo))]
-        (let [files (bean/->clj files)]
-          (p/let [remote-latest-commit (get-remote-ref repo)
-                  local-latest-commit (get-ref repo)
-                  descendent? (git/descendent? repo local-latest-commit remote-latest-commit)
-                  diffs (git/get-diffs repo local-latest-commit remote-latest-commit)]
-            (let [files (if descendent?
-                          (->> (concat (map :path diffs) files)
-                               distinct)
-                          files)]
-              (state/set-changed-files! repo files)))))
+      (p/let [files (js/window.workerThread.getChangedFiles (util/get-repo-dir repo))
+              files (bean/->clj files)]
+        (->
+         (p/let [remote-latest-commit (get-remote-ref repo)
+                 local-latest-commit (get-ref repo)]
+           (p/let [descendent? (git/descendent? repo local-latest-commit remote-latest-commit)
+                   diffs (git/get-diffs repo local-latest-commit remote-latest-commit)]
+             (let [files (if descendent?
+                           (->> (concat (map :path diffs) files)
+                                distinct)
+                           files)]
+               (state/set-changed-files! repo files))))
+         (p/catch (fn [error]
+                    (log/warn :git/ref-not-found {:error error})))))
```

Comment:
It means the remote ref is not found, is it right?

Reply:
Yes.